packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/chuckAfterLineVest.yml

```diff
@@ -1,7 +1,7 @@
 languageId: plaintext
 command:
   version: 1
-  spokenForm: chuck after line vest
+  spokenForm: chuck trailing line vest
```

Comment:
And this one. Was there some migration we did where "chuck after" became "chuck trailing"?

Reply:
Yes
https://github.com/cursorless-dev/cursorless/blob/6a78771b47308a8574a0fa46a50723c0f2c936e9/packages/cursorless-engine/src/core/commandVersionUpgrades/upgradeV1ToV2/upgradeV1ToV2.ts#L108-L114